.npmignore

```diff
@@ -9,3 +10,12 @@ src/ios/.framework_version
 RELEASING.md
 jest.config.js
 examples/
+apiTester/
+Dangerfile
+.version
+.npmrc.SAMPLE
+fastlane/
```

Comment:
I noticed we were adding a bunch of unnecessary files to the resulting package. I added a few of these to not include as many.